WalletWasabi/WabiSabi/Models/DecompositionAlgs/GreedyDecomposer.cs

```diff
@@ -0,0 +1,67 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace WalletWasabi.WabiSabi.Models.DecompositionAlgs
+{
+	public class GreedyDecomposer
+	{
+		public GreedyDecomposer(IEnumerable<Money> preferredDenominations, Money dustThreshold, FeeRate feeRate)
+		{
+			Decomposition = new(preferredDenominations);
```

Comment:
this is not supposed to be the decomposition, but the set of allowed denomination with which the decomposition can be created

Reply:
I fixed the concept.